lib/plugins/aws/package/index.js

```diff
@@ -102,6 +117,11 @@ class AwsPackage {
         await this.saveServiceState();
         return this.serverless.pluginManager.spawn('aws:common:moveArtifactsToPackage');
       },
+
+      'after:aws:package:finalize:saveServiceState': () => {
```

Comment:
I'm a bit on the fence here - I was thinking about adding a new lifecycle hook that runs last instead of attaching with `after` - what is your opinion @medikoo ?

Reply:
Yes, maybe let's add `finalize` lifecycle hook (sort of counterpart of `initialize`), it'll be run only in case of command success (otherwise we have `error`)